src/interp_routines.F

```diff
@@ -576,10 +590,13 @@ subroutine vinterp_flx5(stag,ix,jy,kz,c1,c2,rrw,dumz,a)
           dumz(i,j,k) = flx3(a(i,j,k+1),a(i,j,k  ),a(i,j,k-1))
         endif
       enddo
+    ENDDO
 
       !----
 
-      k = 2
+    k = 2
+    !$acc parallel loop gang vector collapse(2) default(present)
```

Comment:
Do we need OpenMP directive here?

Reply:
Yes we do, I have added in OpenMP directives